src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -1030,4 +1033,17 @@ public static void createSinkBQExistingDatatypeTable() throws IOException, Inter
     PluginPropertyUtils.addPluginProp(" bqTargetTable",  bqTargetTable);
     BeforeActions.scenario.write("BQ Target Table " +  bqTargetTable + " updated successfully");
   }
+  private static String createGCSBucketWithFileAndLifeCycle(String filePath) throws IOException, URISyntaxException {
+    String bucketName = StorageClient.createBucketwithLifeCycle("00000000-e2e-" + UUID.randomUUID(),5 ).getName();
+    StorageClient.uploadObject(bucketName, filePath, filePath);
+    PluginPropertyUtils.addPluginProp("gcsSourceBucketName", bucketName);
+    PluginPropertyUtils.addPluginProp("gcsSourcePath", "gs://" + bucketName + "/" + filePath);
+    BeforeActions.scenario.write("Created GCS Bucket " + bucketName + " containing " + filePath + " file");
+    return bucketName;
+  }
+
+  @Before(order = 1, value = "@GCS_CSV_MULTI_PART_UPLOAD")
```

Comment:
replace CSV text with SINK text in hook name

Reply:
Changes Added as per the Suggestion.